Core/src/main/java/net/juligames/core/command/inbuild/InbuiltCommand.java

```diff
@@ -0,0 +1,94 @@
+package net.juligames.core.command.inbuild;
+
+import net.juligames.core.Core;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ * @author Ture Bentzin
+ * 17.02.2023
+ */
+@ApiStatus.Experimental
+public class InbuiltCommand implements CommandExecutor {
+    private final String name;
+    private final String description;
```

Comment:
I think the description is currently not interpreted, handled or displayed by the InbuiltCommandManager. I plan to make then readable through hazelcast or though the client itself, maybe i should rewrite the command logic and implement a command input to every core. This way i could allow core Commands to run on all cores. This would also demolish the MasterCommand system, that is still in use today. It was only meant for testing purposes. This CommandSystem could be implemented using Brigadier... I will to investigation on how to implement such a system and on how to allow plugins to register their own commands. Maybe i should pull the idea of a information hazelcast object for every client out of the list for 2.0 and implement it earlier...